Zend/tests/type_declarations/typed_properties_110.phpt

```diff
@@ -0,0 +1,13 @@
+--TEST--
+Test typed properties allow null
+--FILE--
+<?php
+class Foo {
+    public null $int;
```

Comment:
```suggestion
    public null $value;
```

nit: Strange (copied?) choice of name

Reply:
Yeah, copied from the 109 test case for never which used $int.